src/tools/tests/goldens/generated-schemas.jvm.kt

```diff
@@ -107,6 +111,131 @@ typealias Gold_Data_Ref_Spec = GoldInternal1_Spec
 typealias Gold_Alias = GoldInternal1
 typealias Gold_Alias_Spec = GoldInternal1_Spec
 
+class Gold_QCollection() : Entity {
+
+    override var internalId = ""
+
+    var name = ""
+        get() = field
+        private set(_value) {
+            field = _value
+        }
+    var age = 0.0
+        get() = field
+        private set(_value) {
+            field = _value
+        }
+    var lastCall = 0.0
+        get() = field
+        private set(_value) {
+            field = _value
+        }
+
+    constructor(
+        name: String = "",
+        age: Double = 0.0,
+        lastCall: Double = 0.0
+    ) : this() {
+        this.name = name
+        this.age = age
+        this.lastCall = lastCall
+    }
+
+    fun copy(
+        name: String = this.name,
+        age: Double = this.age,
+        lastCall: Double = this.lastCall
+    ) = Gold_QCollection(
+        name = name,
+        age = age,
+        lastCall = lastCall
+    )
+
+    fun reset() {
+        name = ""
+        age = 0.0
+        lastCall = 0.0
+    }
+
+    override fun equals(other: Any?): Boolean {
+      if (this === other) {
+        return true
+      }
+
+      if (other is Gold_QCollection) {
+        if (internalId != "") {
+          return internalId == other.internalId
+        }
+        return toString() == other.toString()
+      }
+      return false;
+    }
+
+    override fun hashCode(): Int =
+      if (internalId != "") internalId.hashCode() else toString().hashCode()
+
+    override fun schemaHash() = "f72d6bee9c5b13d2133e2af89a5ed591d670ee74"
+
+    override fun serialize() = RawEntity(
+        internalId,
+        mapOf(
+            "name" to name.toReferencable(),
+            "age" to age.toReferencable(),
+            "lastCall" to lastCall.toReferencable()
+        )
+    )
+
+    override fun toString() = "Gold_QCollection(name = $name, age = $age, lastCall = $lastCall)"
+}
+
+class Gold_QCollection_Spec() : EntitySpec<Gold_QCollection> {
+
+    companion object {
+        val schema = Schema(
```

Comment:
if this needs to be a public constant, its name should be all caps

Reply:
I uppercased it but I'd really appreciate it we did some code generation clean up separate to this change.